alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsSourceMapsAction.class.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @package Core
+ * @subpackage externalWidgets
+ */
+class embedPlaykitJsSourceMapsAction extends sfAction
+{
+    var $uiconf_id = null;
+
+    public function execute()
+    {
+        //Get file name
+        $fileName = $this->getRequestParameter('path');
+        if (!$fileName)
+            KExternalErrors::dieError(KExternalErrors::MISSING_PARAMETER, 'path');
+        $bundleWebDirPath = kConf::get('playkit_js_bundles_path');
+        $sourceMapFilePath = $bundleWebDirPath .$fileName ;
+        $sourceMap = file_get_contents($sourceMapFilePath);
```

Comment:
This code is problematic from security POV.
condsider that path contains- /WEBDIRPATH/../../../<ANY PATH in the server>
Need to make sure that path does not contain ..